git-odb/src/traits.rs

```diff
@@ -45,7 +45,7 @@ pub trait Find {
     /// Returns true if the object exists in the database.
     fn contains(&self, id: impl AsRef<git_hash::oid>) -> bool;
 
-    /// Find an object matching `id` in the database while placing its raw, undecoded data into `buffer`.
+    /// Find an object matching `id` in the database while placing its raw, encoded data into `buffer`.
```

Comment:
This isn't correct - it should remain `unencoded` or something semantically similar, not opposite.

Reply:
Hi, sorry about that, I think the confusion came from that it's originally "un*de*coded", not "un*en*coded". So I think either way there was a typo? Either way, since both undecoded and unencoded aren't actually words, I think it would be better to use their positive counterpoint, encoded and decided respectively.